src/ncp/spinel.c

```diff
@@ -105,13 +105,21 @@ static size_t spinel_strnlen_(const char *s, size_t maxlen)
 #endif
 
 #ifndef require_action
+#if SPINEL_PLATFORM_SHOULD_LOG_ASSERTS
```

Comment:
Is there a doc to captures special compile-time flags, defaults, and the resultant behaviors?